nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -0,0 +1,105 @@
+
+    def handle(self, *args, **options):
+        """Handle execution of the import_netbox_json management command."""
+
+        def process_objectchange(entry):
+            """Processes on objectchange entry to migrate from Netbox to Nautobot."""
+            try:
+                entry["fields"]["changed_object_type"] = ContentType.objects.filter(
+                    id=nautobot_contenttype_mapping[netbox_contenttype_mapping[entry["fields"]["changed_object_type"]]]
+                ).first()
+            except KeyError:
+                logger.warning(
+                    f'{netbox_contenttype_mapping[entry["fields"]["changed_object_type"]]} key is not mapped'
+                )
+                return
+
+            if entry["fields"]["related_object_type"]:
+                entry["fields"]["related_object_type"] = ContentType.objects.filter(
+                    id=nautobot_contenttype_mapping[netbox_contenttype_mapping[entry["fields"]["related_object_type"]]]
+                ).first()
+            else:
+                del entry["fields"]["related_object_type"]
+
+            modelname = entry["fields"]["changed_object_type"].model
+            entry["fields"]["changed_object_id"] = netbox_pk_to_nautobot_pk(
+                modelname, entry["fields"]["changed_object_id"]
+            )
+            if entry["fields"]["related_object_id"]:
+                modelname = entry["fields"]["related_object_type"].model
+                entry["fields"]["related_object_id"] = netbox_pk_to_nautobot_pk(
+                    modelname, entry["fields"]["related_object_id"]
+                )
+
+            entry["fields"]["user"] = User.objects.filter(username=entry["fields"]["user_name"]).first()
+            if not entry["fields"]["user"]:
+                logger.error(f'Username {entry["fields"]["user_name"]} not present in DB.')
+                return
+
+            if not options["dryrun"]:
+                obj = ObjectChange.objects.create(**entry["fields"])
+                obj.time = entry["fields"]["time"]
+                obj.save()
+
+        validate_netbox_version(options["netbox_version"])
+
+        logger, _ = initialize_logger(options)
+
+        logger.info("Loading NetBox JSON data into memory...", filename={options["json_file"].name})
+        no_objectchange_data = json.load(options["json_file"])
+
+        if not isinstance(no_objectchange_data, list):
+            raise CommandError(f"Data should be a list of records, but instead is {type(no_objectchange_data)}!")
+        logger.info("JSON data loaded into memory successfully.")
+
+        logger.info("Creating NetBox ContentType mapping...")
+        netbox_contenttype_mapping = {}
+        for entry in no_objectchange_data:
+            if entry["model"] == "contenttypes.contenttype":
+                netbox_contenttype_mapping[entry["pk"]] = (
+                    entry["fields"]["app_label"],
+                    entry["fields"]["model"],
+                )
```

Comment:
Might be an idea to free the data here by setting `no_objectchange_data = None` after we're done processing it?